adafruit_stmpe610.py

```diff
@@ -235,24 +244,24 @@ def get_version(self):
         return version
 
     @property
-    def touched(self):
+    def touched(self) -> bool:
         """Report if any touches were detected."""
         touch = self._read_byte(_STMPE_TSC_CTRL) & 0x80
         return touch == 0x80
 
     @property
-    def buffer_size(self):
+    def buffer_size(self) -> int:
         """The amount of touch data in the buffer."""
         return self._read_byte(_STMPE_FIFO_SIZE)
 
     @property
-    def buffer_empty(self):
+    def buffer_empty(self) -> bool:
         """Buffer empty status."""
         empty = self._read_byte(_STMPE_FIFO_STA) & _STMPE_FIFO_STA_EMPTY
         return empty != 0
 
     @property
-    def get_point(self):
+    def get_point(self) -> dict:
```

Comment:
We can better specify the contents of the dictionary using generics; let's type this as `Dict[str, int]`.